pkg/server/controllers.go

```diff
@@ -214,19 +208,17 @@ func (s *Server) installKubeServiceAccountTokenController(ctx context.Context, c
 		return fmt.Errorf("error creating service account controller: %w", err)
 	}
 
-	s.AddPostStartHook("kcp-start-kube-service-account-token-controller", func(hookContext genericapiserver.PostStartHookContext) error {
+	return s.AddPostStartHook("kcp-start-kube-service-account-token-controller", func(hookContext genericapiserver.PostStartHookContext) error {
```

Comment:
This return pattern seems like another independent change?

Reply:
no. I unified the `Server.AddPostStartHook` signature and lint complained here.